### ISSUE TITLE
New to Blockchain

### ISSUE THREAD

**Issue by j-batey** [gitcoinco/web#8799]
### Discussion

Hello All! I very recently decided to change careers from the construction industry.  I knew I wanted to get into developing but wasn't sure which direction to go.  Landing on your mission page has changed the trajectory of my life. I struggled to figure out what I was truly passionate about aside from my family. Ive come up what I suppose would be called my own mission. I want to be part of a team that leverages cutting edge technology to level the socioeconomic playing field around the world. I want to be part of the revolution.

While I'm not quite job ready with blockchain, I am pretty tech savvy and feel somewhat comfortable navigating through this path after a few weeks of dabbling. As aforementioned I recently left the construction Industry. I was a union ironworker and wore alot of hats.  I worked with some really bright minds that I feel are the type of people I want on my team when I'm trying to change the world. The problem is some are intimidated by blockchain and others think any job involving computers means "I'll be sitting in front of a computer 12 hours a day no way I can do that". 

As we all know the demand for devs and other technical roles alike will continue to soar in the years ahead. My suggestion would be to possibly create a small subdivision of quests aimed at GOOD people with proficient computer skills using "baby steps" as to not overwhelm and deter.  In my personal experience, tasks that seem very difficult can almost feel easy if you can get a couple little wins to get your flow going. I'm not suggesting spoon feeding, rather a predictably enjoyable first 15 minutes of quests to get the intelligent guy with low self confidence engaged and fascinated who might turn away after a few failed quests otherwise.  

Im an infant in this space and feel absurd giving this community a suggestion. BUT I've been told some of the best suggestions actually come from beginners as the material is so fresh for them and old for more experienced folk. At the end of the day I just don't want to see anyone not reach their full potential because of their lack of confidence. 

Thank you for reading my rambling,
Jarrod Batey

**Comment by mattcan** [comment]
Thanks for sharing Jarrod! What are some topics you would have found to be useful in your introductory phase?

**Comment by kosiokostov** [comment]
Jarod, I believe you speak on the behalf of a lot of us.

**Comment by Davejr81** [comment]
Yes Jarod, I’m in the same boat and am new yet ready to start my crypto journey.
